src/sql/sqlops.proposed.d.ts

```diff
@@ -296,15 +296,22 @@ declare module 'sqlops' {
 		Error = 3
 	}
 
+	export enum CardType {
+		actionCard = 'actionCard',
+		selecableCard = 'selectableCard'
```

Comment:
Not too important but I think elsewhere our enum options start with a capital letter